objs/animate.cc

```diff
@@ -59,6 +59,22 @@ static inline bool Get_sfx_out_of_range(
 		   > (MAX_SOUND_FALLOFF * MAX_SOUND_FALLOFF);
 }
 
+static inline bool Is_halt_frame(
```

Comment:
Please run clang-format on this file.

Reply:
I think I ran clang-format correctly. Please let me know if it didn't reformat correctly.